<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>876</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: rgb(37, 37, 38);
color: rgb(255, 255, 255);
}
QWidget .QPushButton {
border:0px;
background-color: rgb(34, 34, 35);
}
QWidget .QPushButton:hover {
border:0px;
background-color: rgb(0, 122, 204);
}

QLineEdit {
border:1px solid rgb(67, 67, 70);
background-color: rgb(51, 51, 55);
color: rgb(255, 255, 255);
}
QLineEdit:disabled {
border:1px solid rgb(67, 67, 70);
background-color: rgb(45, 45, 48);
color: rgb(255, 255, 255);
}

QWidget .QFrame {
background-color: rgb(45, 45, 48);
color: rgb(255, 255, 255);
}
QFrame .QPushButton {
border:1px solid rgb(83, 83, 85);
background-color: rgb(63, 63, 70);
color: rgb(255, 255, 255);
}
QFrame .QPushButton:disabled {
border: 1px solid rgb(61, 61, 67);
background-color: rgb(45, 45, 48);
}
QFrame .QPushButton:hover {
background-color: rgb(63, 63, 70);
border: 1px solid rgb(5, 111, 182);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QLabel" name="label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hbox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="17" column="0" alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="zeroset">
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>KoPub돋움체 Medium</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>영점</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0" alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="btn_scale">
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>KoPub돋움체 Medium</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>스케일 계산</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0" alignment="Qt::AlignHCenter">
           <widget class="QTextEdit" name="scale_txt">
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>KoPub돋움체 Medium</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'KoPub돋움체 Medium'; font-size:20pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;스케일 팩터&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QTextEdit" name="filename">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>KoPub돋움체 Medium</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'KoPub돋움체 Medium'; font-size:20pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:30px;&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;파일명 입력&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_file">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>KoPub돋움체 Medium</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>저장 경로 설정</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="15" column="0" alignment="Qt::AlignHCenter">
           <widget class="QTextEdit" name="disp_max">
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>KoPub돋움체 Medium</family>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::DefaultContextMenu</enum>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'KoPub돋움체 Medium'; font-size:18pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;현재 변위&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="16" column="0" alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="btn_measure">
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>KoPub돋움체 Medium</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>측정 시작/중지</string>
            </property>
           </widget>
          </item>
          <item row="18" column="0" alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="btn_save">
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>KoPub돋움체 Medium</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>저장</string>
            </property>
           </widget>
          </item>
          <item row="13" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QSlider" name="dispRange0">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBelow</enum>
              </property>
              <property name="tickInterval">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="dispRange1">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBelow</enum>
              </property>
              <property name="tickInterval">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0" alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="btn_align">
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>KoPub돋움체 Medium</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>레이저 정렬</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QTextEdit" name="textEdit">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>KoPub돋움체 Medium</family>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'KoPub돋움체 Medium'; font-size:18pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:30px;&quot;&gt;각도 입력&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_ang">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>KoPub돋움체 Medium</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>각도 측정</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="12" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>400</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>KoPub돋움체 Medium</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="text">
               <string>그래프 y축 범위</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>935</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
